nix/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Dockerfile to create an environment that contains the Nix package manager.
+
+FROM alpine as build
+
+# Enable HTTPS support in wget and set nsswitch.conf to make resolution work within containers
+RUN apk add --no-cache --update openssl git \
+  && rm -rf /var/cache/apk/* \
+  && echo hosts: files dns > /etc/nsswitch.conf
+
+# Download Nix and install it into the system.
+ARG NIX_VERSION=2.5.1
```

Comment:
@vdovhanych should we stich to latest 2.7.0?

Reply:
its already on 2.6.0 which should be the stable release I was messing with it yesterday but directly to master until I iron out the setup here.

Reply:
Great 👍